app/enterprise/0.35-x/deployment/installation/centos.md

```diff
@@ -1,216 +1,238 @@
+1. Switch to PostgreSQL user
 
-1. Switch to PostgreSQL user  
     ```
     $ sudo -i -u postgres
     ```
 
 2. Launch PostgreSQL
+
     ```
     $ psql
     ```
 
-3. Create a Kong user and database, make Kong the owner of the database, and 
-set the password of the Kong user to 'kong'
+3. Run the following command to:
+
+    -- Create a Kong user and database
+
+    --  Make Kong the owner of the database
+
+    --  Set the password of the Kong user to 'kong'
+
     ```
     $ CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong';
     ```
 
+    >⚠️**Note**: Make sure the username and password for the Kong Database are
+    >kept safe. We have used a simple example for illustration purposes only.
+
 4. Exit from PostgreSQL
+
     ```
     $ \q
     ```
 
 5. Return to terminal
+
     ```
     $ exit
     ```
 
-6. Modify the PostgreSQL configuration to allow password authentication with 
-MD5.
-    Under IPv4 local connections: replace the `"ident"` entry with `"md5"`
-  
+6. Run the following command to access the PostgreSQL configuration file.
+
     ```
     $ sudo vi /var/lib/pgsql/9.5/data/pg_hba.conf
     ```
 
-7. restart PostgreSQL
+7. Under IPv4 local connections replace `ident` with `md5`
+
+    | TYPE | DATABASE | USER | ADDRESS      | METHOD |
+    | # IPv4 local connections:                      |
+    | host | all      | all  | 127.0.0.1/32 | **md5**|
+    | # IPv6 local connections:                      |
+    | host | all      | all  | ::1/128      | ident  |
+
+8. Restart PostgresQL
+
     ```
     $ sudo systemctl restart postgresql-9.5
     ```
 
-8. Copy the license file from your home directory to the `/etc/kong` directory. 
-Kong will look for the license here. 
-    ```
-    $ sudo cp kong-se-license.json /etc/kong/license.json
-    ```
 
-9. Modify the `kong.conf` file and update the PostgreSQL database password to 
-the password you used when you created the PostgreSQL user named **kong**. 
+## Step 5. Modify Kong's configuration file
+
+To use the newly provisioned PostgreSQL database, Kong's configuration file
+must be modified to accept the correct PostgreSQL user and password.
+
+1. Make a copy of the default configuration file
 
-10. Make a copy of the default configuration file    
     ```
     $ cp /etc/kong/kong.conf.default /etc/kong/kong.conf
     ```
 
-11. Uncomment and add 'kong' to pg_password line
+2. Uncomment and update the PostgresQL database properties inside the Kong conf:
+
     ```
     $ sudo vi [/path/to/kong.conf]
     ```
+    ```
+    pg_user = kong
+    pg_password = kong
+    pg_database = kong
+    ```
 
-## Step 6 
 
-Seed the **Super Admin** for initial startup. This step is not mandatory, 
-but if you want to add security and Role-Based Access Control (RBAC) to your 
-environment, it is best to seed the **Super Admin** at this step and make note 
-of the password for when you need it later. 
+## Step 6. Seed the **Super Admin** *(optional)*
```

Comment:
bold doesn't render in a title, italics will. This suggestion assumes that (optional) was the point to emphasize.

```suggestion
## Step 6. Seed the Super Admin _(optional)_
```